e2e/app/page/authenticated-page.ts

```diff
@@ -21,9 +20,13 @@ export default abstract class AuthenticatedPage extends BasePage {
   }
 
   protected async isSignedIn(): Promise<boolean> {
-    await this.page.waitForSelector(SELECTOR.signedInIndicator);
-    await this.page.waitForSelector(SELECTOR.logo, {visible: true});
-    return true;
+    try {
+      await this.page.waitForSelector(SELECTOR.signedInIndicator);
+      await this.page.waitForSelector(SELECTOR.logo, {visible: true});
```

Comment:
The AoU logo is visible on signed out/account creation pages as well

Reply:
you're correctly. I updated `signedInIndicator ` to be `app-signed-in`.